opacus/privacy_engine.py

```diff
@@ -207,6 +203,9 @@ def _prepare_model(
         try_fix_incompatible_modules: bool = False,
```

Comment:
do we want to remove `try_fix_incompatible_modules` from here as well?

Reply:
I kept it as is since, technically the user can call `privacy_engine._prepare_model()` with the flag. However, the method is meant to be private, so it doesn't make sense anyway. I'll remove it